docs/source/system_replication/system_replication.rst

```diff
@@ -0,0 +1,114 @@
+==================================================
+USB-Based System Replication
+==================================================
+
+.. contents:: Table of Contents
+   :depth: 2
+   :local:
+
+Introduction
+============
+
+The `USB-based system replication` feature allows saving the full disk image of an x64
+based NI Linux RT capable target and applying it to same or different target(s) of the
+same model.
+
+Safemode, runmode contents, drivers, can all be replicated.
+
+Some settings such as Hostname, SSH keys, Salt Minion ID, that uniquely identify a
+device on a network are not replicated to prevent conflicts on the network.
+
+Creating USB drive
+==================
+
+NI Linux RT System Image ships with a `nilrt-recovery-media-x64.iso`. Version 24.8 and
+above can be used to create a USB drive capable of system replication.
```

Comment:
The NI Linux RT System Image includes a disk image file named `nilrt-recovery-media-x64.iso`. You can use this disk image file to prepare a USB drive in order to perform various operations on NI Linux RT targets. NI Linux RT System Image versions 24.8 and above include a version of this file that includes system replication functionality.

Reply:
Reworded original text and made it a note at end of section and reworded. Let me know if that isn't ok.